### ISSUE TITLE
Need a form for register new user

### ISSUE THREAD

**Issue by sonit1810** [divppc/Physio-App#29]
Please clone this form with new style into one page : register.html

https://myphysioapp.com.au/users/register.jsp?from=trial

**Comment by divppc** [comment]
you need new page with this content and new design, right?

**Comment by sonit1810** [comment]
Yes Ihor,  but leave this for now. Please focus to other issues.

**Comment by sonit1810** [comment]
Hi Ihor, please do it after other issues, thanks!

**Comment by sonit1810** [comment]
It's new page, thanks!
